tidb-cloud/tidb-cloud-clinic.md

```diff
@@ -0,0 +1,109 @@
+---
+title: TiDB Cloud Clinic
+summary: Learn how to advanced monitoring and diagnostics of TiDB Cloud.
+---
+
+# TiDB Cloud Clinic
+
+TiDB Cloud Clinic offers advanced monitoring and diagnostic capabilities on the TiDB Cloud.
+
+> **Note:**
+>
+> Currently, the TiDB Cloud Clinic feature is only available for [TiDB Cloud Dedicated](/tidb-cloud/select-cluster-tier.md#tidb-cloud-dedicated) clusters.
+
+## Prerequisites
+
+To use TiDB Cloud Clinic, you need to have an Enterprise support plan or higher on TiDB Cloud.
+
+## View the Cluster page
+
+To view the Cluster page, take the following steps:
+
+1. Log in to the [TiDB Cloud Clinic console](https://clinic.pingcap.com/)
+
+2. Click the name of the target orgnization. The clusters in one project are displayed.
+
+3. Click the name of the target cluster. The cluster overview page is displayed.
+
+
+- Advanced Metrics
+- Top Slow Queries
+- TopSQL
+- Benchmark Report
+
+### Advanced Metrics
+
+TiDB Cloud Clinic uses Grafana to provide a wider range of TiDB cluster metrics. These include dashboards for TiDB, TiKV, TiFlash, BR, Lightning, TiCDC, TiDB resource control, and TiDB performance.
+
+To view the metrics dashboard, take the following steps:
+
+1. Navigate to the **Cluseter** page of a cluster.
+
+2. Click the **Metrics** button.
+
+3. Click the name of target dashboard. The dashboard is displayed.
+
+The retention policy for advanced metrics is 90 days.
+
+### Top Slow Queries
+
+By default, SQL queries that take more than 300 milliseconds are considered slow queries.
+
+On the default Slow Queries page, it can be challenging to quickly identify performance-impacting SQL queries when a cluster has a large volume of slow queries. Top Slow Queries provides aggregated analysis based on slow query logs. With this feature, you can easily pinpoint queries with performance issues, reducing overall performance tuning time by at least half.
```

Comment:
```suggestion
By default, SQL queries that take longer than 300 milliseconds are considered slow queries.

On the default **Slow Queries** page, it can be difficult to quickly identify performance-impacting SQL queries when a cluster has a large volume of slow queries. The **Top Slow Queries** feature provides aggregated analysis based on slow query logs. With this feature, you can easily pinpoint queries with performance issues, reducing overall performance tuning time by at least half.
```